kaldb/src/main/proto/trace.proto

```diff
@@ -29,8 +31,10 @@ message KeyValue {
   string    v_str     = 3;
   bool      v_bool    = 4;
   int64     v_int64   = 5;
+  int32     v_int32   = 9;
   double    v_float64 = 6;
-  bytes     v_binary  = 7;
+  float     v_float32 = 10;
+  slack.proto.kaldb.schema.SchemaFieldType fieldType = 8;
```

Comment:
Isn't schema a property of the dataset? If so, why is it in a `KeyValue` type?

Reply:
Schema will be part of the dataset. That will continue to be in the indexer. Here we just want to inject field type info from the preprocessor that the indexer can leverage.